common/src/main/java/com/microsoft/identity/common/adal/internal/AuthenticationConstants.java

```diff
@@ -1247,6 +1257,50 @@ private AuthenticationConstants() {
         public static final String REMOVE_ACCOUNT_FROM_SHARED_DEVICE = "REMOVE_ACCOUNT_FROM_SHARED_DEVICE";
     }
 
+
+    public static final class BrokerContentProvider{
```

Comment:
If this is a constant collection, and it looks like it is, should it have a private constructor?  The other instances of this have documentation describing each constant, should this?  Is there a link to documentation that could be referenced?

Reply:
Addressed via https://github.com/AzureAD/microsoft-authentication-library-common-for-android/pull/918/commits/477877f2d99d6eb7d752f18d941a682b51665c62